.github/workflows/tests.yaml

```diff
@@ -18,10 +18,10 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v1
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
     - uses: actions/cache@v1
```

Comment:
Might want to consider updating checkouts, setup-java, and cache to v3 since this stuff is already getting updated.

Reply:
didn't you work an effort to update that across the board for all of our projects?  any reason this one was left out?  I imagine we want this on main as well, right?

Reply:
I have started PR's on each microservice to update these.  DataWave mono repo has already been updated and merged.

Reply:
If you're still working that effort then I'll be happy to merge in your changes for this repo once you make them.